src/Ajax.php

```diff
@@ -0,0 +1,157 @@
+<?php
+namespace Krokedil\SignInWithKlarna;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+require_once 'includes/functions.php';
```

Comment:
You should not have a require_once like this. Id rather see a class that handles the JWT stuff in this case